### ISSUE TITLE
MPL-2.0 exhibit B

### ISSUE THREAD

**Issue by spwhitton** [pikepdf/pikepdf#8]
I am trying to understand what is meant by your removal of Exhibit B from the copies of the MPL that are included in the pikepdf repository.

Firstly, "Exhibit" means that the text is an example, so removing it from the license is not any kind of declaration about your code.

Secondly, you have removed it from the license without removing references to it from the license.  So the license does not really make sense anymore.  For example, it still says "If You choose to distribute Source Code Form that is Incompatible With Secondary Licenses under the terms of this version of the License, the notice described in Exhibit B of this License must be attached."

Thirdly, I cannot tell what "We exclude Exhibit B, so pikepdf is compatible with secondary licenses." means.  Do you mean that you exclude the requirement to attach the notice described in Exhibit B?

Could you say what your intention is here, please?  Thanks.

**Comment by jbarlow83** [comment]
The intention was to be *compatible* with secondary licenses. I was under the impression that I had to delete Exhibit B because verbatim, it says the opposite of what I wanted, but I now see this is not the case.

I'll restore Exhibit B to avoid any concerns that arise from having a modified license file. Since MPL 2.0 is already compatible with (L)GPL there is no need to add special remarks authorizing this combination use.

As far as packaging, the Debian Firefox VCS licenses itself under MPL-2.0 as well. In the interest of avoiding future hassle it may be easiest to copy that approach. Otherwise you might need to add a (L)GPL notice (https://www.mozilla.org/en-US/MPL/2.0/combining-mpl-and-gpl/). 

I think this is a reasonable time to be packaging for Debian experimental. ArchLinux packaged pikepdf successfully.

**Comment by spwhitton** [comment]
Thank you for your response.

Yes, I'll license my packaging work under MPL-2.0.

I have finished packaging and just need to write the copyright file, which is why I filed this issue.  My packaging is based on 0.3.0 because the newer release contains the vendored pybind11.  I can work around the vendored copy, but it takes some effort, so since you said the vendoring is temporary I thought I'd just wait until you removed it before packaging a more recent version.  So my plan is to backport your commit restoring Exhibit B to 0.3.0 as a Debian-specific patch.
